x-pack/plugins/security_solution/public/resolver/view/panels/copyable_panel_field.tsx

```diff
@@ -69,15 +92,13 @@ export const CopyablePanelField = memo(
           isOpen={isOpen}
           panelPaddingSize="s"
         >
-          <EuiToolTip
-            content={i18n.translate('xpack.securitySolution.resolver.panel.copyToClipboard', {
-              defaultMessage: 'Copy to Clipboard',
-            })}
-          >
-            <WithCopyToClipboard
-              data-test-subj="resolver:panel:copy-to-clipboard"
-              text={textToCopy}
-              titleSummary={textToCopy}
+          <EuiToolTip content={TRANSLATED_COPY_TO_CLIPBOARD}>
+            <EuiButtonIcon
+              aria-label={TRANSLATED_COPY_TO_CLIPBOARD}
+              color="text"
+              data-test-subj="clipboard"
```

Comment:
Is this `data-test-subj` correct? In the old code it was namespaced under `resolver:...`

Reply:
Thanks, yea, I fixed it.